src/components/InfoComponents/CafeteriaMenu.js

```diff
@@ -346,6 +353,7 @@ export default function CafeteriaMenu(
           </MobileCafeteria>
         </MobileCafeteriaMenu>
       </CafeteriaContainer>
+        {cafeteriaMenus[0]?.updated_at ? <Update>{"최신 업데이트 날짜 : "+cafeteriaMenus[0]?.updated_at.slice(0, 11)}</Update> : <Update></Update>}
```

Comment:
메뉴에는 반드시 updated_at이 포함되어있으므로, `cafeteriaMenus[0]` 만으로 비교해도 괜찮을 것 같아요

Reply:
그리고 Update라는 태그가 반드시 포함되니깐 태그를 중복해서 쓰지 않아도 될거같아요
```suggestion
        <Update>
          {cafeteriaMenus[0]? `최신 업데이트 날짜 : ${cafeteriaMenus[0]?.updated_at.slice(0, 11)}}` : ''}
        <Update>
```

Reply:
수정했습니다!
d0d1f30a97eec1eb13e0421cfa0ffb95c7b5240f